src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -37,11 +37,15 @@ class OgSelection extends DefaultSelection {
    * @return DefaultSelection
    */
   public function getSelectionHandler() {
-    $field = $this->getConfiguration('field');
-
-    if ($field) {
-      return \Drupal::service('plugin.manager.entity_reference_selection')->getSelectionHandler($field);
```

Comment:
Currently, if you try to edit some group content, try to use the autocompletion to find another group, you will not get anything.